File: peel-core/src/main/java/eu/stratosphere/peel/core/beans/StringToConfig.java
package eu.stratosphere.peel.core.beans;

import org.springframework.core.convert.converter.Converter;

import com.typesafe.config.Config;
import com.typesafe.config.ConfigFactory;

public class StringToConfig implements Converter<String, Config> {

    @Override
    public Config convert(String s) {
        return ConfigFactory.parseString(s);
    }
}

File: peel-core/src/main/scala/eu/stratosphere/peel/core/beans/StringToConfig.scala
package eu.stratosphere.peel.core.beans

import com.typesafe.config.{ConfigFactory, Config}
import org.springframework.core.convert.converter.Converter

class StringToConfig extends Converter[java.lang.String, Config] {

  def convert(s: java.lang.String): Config = {
    ConfigFactory.parseString(s)
  }
}
